docs/METRICS.md

```diff
@@ -158,6 +158,16 @@ Below is a sample ping for the `badge_toolbar_button` and `visit_supported_site`
   - `'product_card'`: Sends the user to the product page for the given Product Card.
   - `'system_notification'`: Sends the user to the product page for the Price Alert displayed in the notification.
   - `'walmart_link'`: Sends the user to Walmart.
+- `'do_not_track'`: 'true' if the user has requested not to be tracked by websites, content, or advertising; otherwise 'false'.
+- `'privacy_tp'`: The user's tracking protection setting:
+  - `'always'`: Tracking Protection is on
+  - `'never'`: Tracking Protection is off
+  - `'private_browsing'`: Tracking Protection is on in private browsing windows only
+- `'cookie_config'`: The user's cookie setting:
```

Comment:
For all of these, I would link at least once to the associated SUMO page if available. Ex: [Tracking Protection](https://support.mozilla.org/en-US/kb/tracking-protection).

Reply:
Why the SUMO page? This document already assumes a pretty thorough understanding of how Firefox works.

Reply:
Well, my thinking is that in the same way that we have a Definitions section for extension-specific concepts, we should also link out to existing, official definitions for Firefox-specific concepts.

You're right that the primary consumers of this doc are Mozilla employees, but it just seems like an easy step to link out to more information for anyone (technical or otherwise) who's not familiar with these Firefox terms/features.